osx/setup.sh

```diff
@@ -349,7 +349,7 @@ fi
 # sign the .app file
 # unlock the keychain first so we can access the private key
 # security unlock-keychain -p $KEYCHAIN_PASSWORD
-# codesign -s "$SIGNER_IDENTITY_APPLICATION" --force "$KA_LITE_MONITOR_APP_PATH"
+codesign -s "$SIGNER_IDENTITY_APPLICATION" --force "$KA_LITE_MONITOR_APP_PATH"
```

Comment:
Try to build with this as the only change (revert the changes above) - see if it succeeds.